components/progress-indicator/private/step-vertical.jsx

```diff
@@ -90,6 +90,32 @@ class StepVertical extends React.Component {
 		);
 	};
 
+	renderStepContent = () => {
+		if (this.props.step.onRenderSetupAssistantAction) {
+			return (
+				<div
+					id={`progress-indicator-vertical-label-${this.props.step.id ||
+						this.props.index}`}
+					className="slds-progress__item_content slds-grid slds-grid_align-spread"
+				>
+					<div className="slds-size_3-of-4">{this.props.step.label}</div>
+					<div className="slds-grid slds-grid_align-end slds-size_1-of-4">
+						{this.props.step.onRenderSetupAssistantAction}
+					</div>
```

Comment:
Here <p class="slds-text-align_right slds-text-color_weak slds-p-top_medium">{prop}</p>

Reply:
Sure, I will put it here.

Reply:
<img width="788" alt="Screenshot 2019-05-04 at 1 17 26 AM" src="https://user-images.githubusercontent.com/24983958/57161934-6e411c80-6e0a-11e9-8c79-351cbf78e462.png">

@interactivellama Does this look fine?